src/routes/(user)/clients/+page.svelte

```diff
@@ -1,11 +1,40 @@
 <script lang="ts">
+	import { goto } from '$app/navigation';
 	import type { PageData } from './$types';
-
 	export let data: PageData;
-</script>
+
+	const goToAddClient = () => {
+		goto('clients/addClient');
+	};
+	</script>
 
 <section class="flex-1 space-y-4 p-8 pt-6">
 	<div class="flex items-center justify-between space-y-2">
 		<h2 class="text-3xl font-bold tracking-tight">Clients</h2>
+		<button type="submit" class="bg-blue-500 hover:bg-blue-700 text-white font-bold py-2 px-4 rounded" on:click={goToAddClient}>Add client</button>
```

Comment:
This should be a normal button. We can replace this with the shadcn-svelte provided Button component.